README.md

```diff
@@ -1 +1,52 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+
+> My First Microverse Project.
+
+
+## Built With
+
+- HTML $ CSS
+
+## Live Demo
+
+- Not available yet
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Clone the repository
+
+### cd into the repository
+
+### Run 'Npm Insttall' to install all the project dependences
+
```

Comment:
- [OPTIONAL] Kindly give a descriptive information on how to clone and use your app locally.